homeassistant/components/fireservicerota/translations/nl.json

```diff
@@ -0,0 +1,24 @@
+{
+    "config": {
```

Comment:
These files are getting overridden by the data in Lokalise, which is an automated process. So unless it's also added to Lokalise, this fill will be automatically deleted.

Reply:
If I remove these files (nl.json and en.json) I see the raw references while adding the integration in the GUI, and when I run 'python3 -m script.translations develop' the en.json gets filled with the raw references, with same result, what do I do wrong/miss?

Reply:
Nothing. It's fine to use it for testing. Just that you should not expect these translations to persist and that Lokalise is the source of truth for non-English translations.